portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsFieldsGroup.testcase

```diff
@@ -122,14 +122,54 @@ definition {
 	}
 
 	@description = "Verify that a Fields Group can be created from an existing Field"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanBeCreatedFromExistingField {
-		property forms.functional = "false";
+		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133878 CanBeCreatedFromExistingField pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.addTextField(fieldPositionNumber = "0");
+
+		FormFields.editPlaceholderField(fieldValue = "Test Placeholder Text");
+
+		FormFields.editHelpTextField(fieldHelpText = "Test Help Text");
+
+		Form.closeSidebar();
+
+		Form.gotoAddField(
+			fieldPositionNumber = "1",
+			fieldType = "Numeric");
+
+		FormFields.editPlaceholderField(fieldValue = "Test Placeholder Text Numeric");
+
+		FormFields.editHelpTextField(fieldHelpText = "Test Help Text Numeric");
+
+		Form.addFieldToFieldsGroup(
+			sourceFieldName = "Numeric",
+			targetFieldLabel = "Text");
+
+		FormFields.viewTextLabel(fieldLabel = "Fields Group");
+
+		Form.viewFormPlaceholderText(key_value = "Test Placeholder Text");
```

Comment:
@wesleyroberts, can we use the macro `FormFields.viewFieldPlaceholderText` instead?

Almost there, @wesleyroberts. :slightly_smiling_face:

Reply:
yes, we can use this macro  FormFields.viewFieldPlaceholderText, change made @natocesarrego